source/_integrations/integration.markdown

```diff
@@ -11,15 +11,42 @@ ha_quality_scale: internal
 ha_codeowners:
   - '@dgomes'
 ha_domain: integration
+ha_config_flow: true
 ha_platforms:
   - sensor
 ---
 
-The `integration` platform provides the [Riemann sum](https://en.wikipedia.org/wiki/Riemann_sum) of the values provided by a source sensor. The Riemann sum is an approximation of an **integral** by a finite sum. The integration sensors is updated upon changes of the **source**. Fast sampling source sensors provide better results. In this implementation, the default is the Trapezoidal method, but Left and Right methods can optionally be used.
+This integrations provides the [Riemann sum](https://en.wikipedia.org/wiki/Riemann_sum)
+of the values provided by a source sensor. The Riemann sum is an approximation
+of an **integral** by a finite sum.
 
-## Configuration
+The integration sensors are updated upon changes of the **source**. Fast
+sampling source sensors provide better results. In this implementation, the
+default is the Trapezoidal method, but Left and Right methods can optionally
```

Comment:
```suggestion
default is the Trapezoidal rule, but Left and Right Riemann sums can optionally
```

Reply:
it's not a rule... its a method https://en.wikipedia.org/wiki/Riemann_sum

Reply:
🤷

Reply:
The article lists four methods: Left Riemann sum, Right Riemann sum, Midpoint rule and Trapezoidal rule

Reply:
<https://en.wikipedia.org/wiki/Trapezoidal_rule>

Reply:
https://en.wikipedia.org/wiki/Trapezoidal_method

🤷